src/proto/grpc/testing/control.proto

```diff
@@ -265,6 +265,10 @@ message ScenarioResultSummary
   // Queries per CPU-sec over all servers or clients
   double server_queries_per_cpu_sec = 17;
   double client_queries_per_cpu_sec = 18;
+
+  // Start and end time for the test scenario
+  int64 start = 19;
```

Comment:
The reason I made the time epoch is because the prometheus range query used the epoch time. It make scene to make them timestamp here so that people can read and convert it in the query script.